definehell.h

```diff
@@ -56,3 +56,9 @@
 
 // variable hell
 #define now =
+
+// type hell

```

Comment:
The hell categories should be put in alphabetical order. For example, type hell would be put between symbol hell and variable hell.